dotcom-rendering/src/web/components/OnwardsUpper.importable.tsx

```diff
@@ -77,36 +77,43 @@ const onwardsWrapper = css`
 	width: 100%;
 `;
 
+// TODO: EUR edition urls are currently pointing to INT edition
+// containers, once the europe edition is live, we may want to change this
 const containerUrls = {
```

Comment:
Could we drive this from FAPI data somehow?

Reply:
Probably not, at least for now - based on my understanding of how this connects at least